src/handler.rs

```diff
@@ -109,7 +109,7 @@ use UpgradeStatus::{NotUpgraded, Upgraded};
 
 fn header_eq_ignore_case<T>(req: &Request<T>, header_name: HeaderName, value: &str) -> bool {
     req.header(header_name)
-        .map(|h| h.as_str().eq_ignore_ascii_case(value))
+        .map(|h| h.as_str().split(",").map(|s| s.trim().eq_ignore_ascii_case(value)).collect::<Vec<bool>>().contains(&true))
```

Comment:
Hi @mu2019, I think this will works, can you change here to `char` as clippy recommend ?

```suggestion
        .map(|h| h.as_str().split(',').map(|s| s.trim().eq_ignore_ascii_case(value)).collect::<Vec<bool>>().contains(&true))
```
Thanks!